packages/components/src/components/icon/currency/ic-currency-xrp.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@mitra-deriv 1st thing - you have to optimize this svg file, you can use https://www.svgviewer.dev/ for this purpose. 2nd, what about dark theme? does it look ok? Or maybe you can pass custom-color to the svg file to control theme.

Reply:
Thanks Sergei for reminding on SVGO , 


Below is Dark theme, I didn't find any design for dark theme 

<img width="958" alt="Screenshot 2024-06-10 at 4 50 10 PM" src="https://github.com/binary-com/deriv-app/assets/64970259/86986826-4da5-41e6-9a9a-dfc749d6d777">

Reply:
@mitra-deriv for me, dark theme looks good :) Just optimize the svg file, and I'll approve your PR

Reply:
Done and thanks @sergei-deriv 🫡